packages/core/src/components/overflow-list/overflowList.tsx

```diff
@@ -85,6 +107,8 @@ export class OverflowList<T> extends React.PureComponent<IOverflowListProps<T>,
     }
 
     public state: IOverflowListState<T> = {
+        direction: OverflowDirection.NONE,
```

Comment:
We initially call repartition in shrinking mode, so maybe this should be `SHRINK` (if you want `onOverflow` to be called initially, even when the overflow is empty. If you don't, just keep `NONE`)

Reply:
i changed to `SHRINK` but it was not called initially in tests, so just sticking with this.